### DISCUSSION TITLE
Instagram Cookies not showing up

### DISCUSSION THREAD

**Discussion by ryphez** [AAndyProgram/SCrawler#185] [Q&A] [answered]
I'm trying to setup to download from instagram and got the csrftoken easily enough but the other 2 ([x-ig-app-id](https://github.com/AAndyProgram/SCrawler/wiki/Settings#how-to-find-headers)
[ix-ig-www-claim](https://github.com/AAndyProgram/SCrawler/wiki/Settings#how-to-find-headers)) don't show up. I'm using Chrome to attempt to get the cookies.

ThankS!

**Comment by AAndyProgram** [comment, marked as answer]
https://github.com/AAndyProgram/SCrawler/wiki/Settings#how-to-find-headers
Use `Ctrl+F` to find the header

**Join our Discord server**: https://discord.gg/uFNUXvFFmg
